charts/wordpress/v9.0.3/values.yaml

```diff
@@ -13,8 +13,8 @@
 ##
 image:
   registry: docker.io
-  repository: ranchercharts/bitnami-wordpress
-  tag: 5.2.3-debian-9-r5
+  repository: bitnami/wordpress
```

Comment:
@loganhz @guangbochen Do you know why we put the images of this chart into `ranchercharts` in Dockerhub?

Reply:
We were trying to use the dockerhub download numbers to see how many times they are deployed from Rancher catalogs

Reply:
@deniseschannon yes, we have mirrored the images to the `ranchercharts` so we can understand the popularity of the catalog app based on the image downloads.
here is the repo that I'm using for the image mirroring. https://github.com/guangbochen/charts-image-mirros
![image](https://user-images.githubusercontent.com/4569037/77148918-75792380-6acb-11ea-922e-3584f2075b4a.png)

Reply:
we have added this on k8s v1.17 migration, but we can either keep doing image-mirrors to track the app downloads or change it back to use the official library.